Ios/3.8.2 (#80)
MapsIndoors/mapsindoors.github.io: collections/_ios/v3/CHANGELOG.md

- Fixed `MPMapControl` is now more resilient against `GMSMapView.delegate` being changed.
- [This issue](https://forums.developer.apple.com/thread/123003) made our SDK crash if built with XCode 10 and below. We have implemented a workaround in this version.
- Fixed Restored previous behaviour where the map settles on a building and showing the floor selector initially.

> - Fixed Improved switching between different solutions / api keys.

Reviewer: ```suggestion
- Fixed Improved switching between different Solutions / API keys.
```

## [3.6.1] 2019-11-05

### Fixed

> - Fixed synchronisation issue, that sometimes caused map graphics to disappear, if the app was killed in the middle of a synchronisation.

Reviewer: ```suggestion
- Fixed synchronisation issue, that sometimes caused map graphics to disappear, if the app was shut down in the middle of a synchronisation.
```

### Changed

> - Updated Google Maps SDK from 3.1.0 to 3.3.0 (see https://developers.google.com/maps/documentation/ios-sdk/releases for details).

Reviewer: ```suggestion
- Updated Google Maps SDK from 3.1.0 to 3.3.0 (see <https://developers.google.com/maps/documentation/ios-sdk/releases> for details).
```
